### PR TITLE
joystick code updated to current commands

### LINKED ISSUE #23: Operator Joystick
- [x] import commands

- [x] a button - full sequence command onTrue
- [x] b - storage backwards on
- [ ] x - acquisition backwards (should also be default command for acquisition but im not 100% sure how to set that) on
- [x] Left button - scoring position (Trap), onTrue
- [x] Down button - scoring position (Ground), onTrue
- [x] Right button - scoring position (Amp), onTrue
- [x] rt trigger - drop note onTrue
- [x] lf trigger - shoot note onTrue

### LINKED ISSUE #77: DriverJoystick
- [x] lift elevate and depress with left and right triggers

### PR DESCRIPTION
**Pull request by matunkar**
### Pull Request

---------------------------------------------

#### Issue Number

Closes #23 
Closes #77

### CODE REVIEW

**Review by SpecialSpark — APPROVED** [review]
I have no qualms with the code and it should run.

File: src/main/java/frc/robot/DriverJoystick.java [hunk 1]
```diff
@@ -88,6 +90,9 @@ private DriverJoystick() {
         super.aButton
                 .onTrue(new InstantCommand(vision::enable0, vision))
                 .onFalse(new InstantCommand(vision::disable0, vision));
+
+        leftTrigger.onTrue(new LiftExtendCommand());
+        rightTrigger.onTrue(new LiftDepressCommand());
```
Comment (reediculous456) [inline src/main/java/frc/robot/DriverJoystick.java:95]: Shouldn't these be wile true? that way if they let go of the button the command will end.
With onTrue, the command won't end until isFinished returns true

File: src/main/java/frc/robot/OperatorJoystick.java [hunk 2]
```diff
@@ -17,5 +29,22 @@ public static OperatorJoystick getInstance() {
...
+                .onTrue(new ScoringPositionCommand(ElevatorSetpoints.Trap));
+
+        new Trigger(() -> operatorJoystick.getPOVPosition() == PovPositions.Down)
+                .onTrue(new ScoringPositionCommand(ElevatorSetpoints.Ground));
+
+        new Trigger(() -> operatorJoystick.getPOVPosition() == PovPositions.Right)
+                .onTrue(new ScoringPositionCommand(ElevatorSetpoints.Amp));
+
+        rightTrigger.onTrue(new ScoreNoteAmpCommand());
+        leftTrigger.onTrue(new ShootNoteCommand());
```
Comment (reediculous456) [inline src/main/java/frc/robot/OperatorJoystick.java:47]: Same here, I think you want while true

**Review by reediculous456 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/DriverJoystick.java [hunk 3]
```diff
@@ -88,6 +90,9 @@ private DriverJoystick() {
         super.aButton
                 .onTrue(new InstantCommand(vision::enable0, vision))
                 .onFalse(new InstantCommand(vision::disable0, vision));
+
+        leftTrigger.onTrue(new LiftExtendCommand());
```
Comment (matunkar) [inline src/main/java/frc/robot/DriverJoystick.java:94]:
```suggestion
        leftTrigger.whileTrue(new LiftExtendCommand());
```

File: src/main/java/frc/robot/DriverJoystick.java (hunk 1 above)
Comment (matunkar) [inline src/main/java/frc/robot/DriverJoystick.java:95]:
```suggestion
        rightTrigger.whileTrue(new LiftDepressCommand());
```

File: src/main/java/frc/robot/OperatorJoystick.java [hunk 4]
```diff
@@ -17,5 +29,22 @@ public static OperatorJoystick getInstance() {
...
+        new Trigger(() -> operatorJoystick.getPOVPosition() == PovPositions.Left)
+                .onTrue(new ScoringPositionCommand(ElevatorSetpoints.Trap));
+
+        new Trigger(() -> operatorJoystick.getPOVPosition() == PovPositions.Down)
+                .onTrue(new ScoringPositionCommand(ElevatorSetpoints.Ground));
+
+        new Trigger(() -> operatorJoystick.getPOVPosition() == PovPositions.Right)
+                .onTrue(new ScoringPositionCommand(ElevatorSetpoints.Amp));
+
+        rightTrigger.onTrue(new ScoreNoteAmpCommand());
```
Comment (matunkar) [inline src/main/java/frc/robot/OperatorJoystick.java:46]:
```suggestion
        rightTrigger.whileTrue(new ScoreNoteAmpCommand());
```

File: src/main/java/frc/robot/OperatorJoystick.java (hunk 2 above)
Comment (matunkar) [inline src/main/java/frc/robot/OperatorJoystick.java:47]:
```suggestion
        leftTrigger.whileTrue(new ShootNoteCommand());
```

**Review by LandisCrawford6114 — APPROVED** [review]

**Pull request merged**